BALSAMIC/containers/vcf2cytosure/Dockerfile

```
@@ -1,28 +1,54 @@
-FROM continuumio/miniconda3:4.10.3-alpine
+FROM python:3.11-slim AS builder
 
-LABEL base.image="continuumio/miniconda3:4.10.3-alpine"
+ARG CONTAINER_NAME
+ENV WORK_DIR /opt/
+ENV VENV ${WORK_DIR}/venv
+
+RUN apt-get update && \
+     apt-get -y upgrade && \
+     apt-get -y install -y --no-install-recommends git tabix && \
+     apt-get clean && \
+     rm -rf /var/lib/apt/lists/*
```

This is my OCD talking, but let's be consistent with the tabs: `tabstop=4`  (and same thing below)
```suggestion
RUN apt-get update && \
    apt-get -y upgrade && \
    apt-get -y install -y --no-install-recommends git tabix && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```